decidim-elections/app/models/decidim/elections/question.rb

```diff
@@ -63,16 +63,9 @@ def published_results?
       end
 
       def publishable_results?
-        return false if published_results?
-
-        case election.results_availability
-        when "per_question"
-          voting_enabled?
-        when "after_end"
-          election.ready_to_publish_results?
-        else
-          false
-        end
+        return false if published_results? || !election.per_question?
+
+        voting_enabled?
```

Comment:
is not this equivalent ?

```suggestion
        published_results?  && election.per_question? && voting_enabled?
```

Reply:
okj